Tests/web3swiftTests/localTests/LocalTestCase.swift

```diff
@@ -8,39 +8,31 @@ import web3swift
 // SuperClass that all local tests should be derived from
 // while this class does show up in the navigator, it has no associated tests
 class LocalTestCase: XCTestCase {
-    static let url = URL.init(string: "http://127.0.0.1:8545")!
-    static var isSetUp = false
 
-    override class func setUp() {
-        super.setUp()
+    static let url = URL(string: "http://127.0.0.1:8545")!
 
-        Task {
-            // check to see if we need to run the one-time setup
-            if isSetUp { return }
-            isSetUp = true
+    override func setUp() async throws {
```

Comment:
@yaroslavyaroslav This is an important fix. 
Because of the nature of async/await the blockchain instance that is deployed during testing doesn't get populated with new transactions **before** the actual test is executed - it all happened in parallel.
That led to CI failing because of gas estimation failure, which failed because of oracle failing to get the gas price in `suggestGasFeeLegacy()`.

So now, before each test this function will run, but now it's synchronous.
So when the first test function will be executed by CI it will populate the blockchain instance with transactions, and the next test case will not add new transactions because of the condition on the line 18:
```swift
if block >= 25 { return }
```